components/browser/menu/src/main/java/mozilla/components/browser/menu/WebExtensionBrowserMenuBuilder.kt

```diff
@@ -22,8 +22,8 @@ import mozilla.components.browser.state.store.BrowserStore
  * @param webExtIconTintColorResource Optional ID of color resource to tint the icons of back and
  * add-ons manager menu items.
  * @param onAddonsManagerTapped Callback to be invoked when add-ons manager menu item is selected.
- * @param appendExtensionActionAtStart true if web extensions appear at the top (start) of the menu,
```

Comment:
Looks like you might have squashed your changes with another one?

Reply:
Ah, good catch, I somehow missed that. the squash should be be gone now 😄